docker/build_artifacts/sagemaker/python_service.py

```diff
@@ -276,8 +296,10 @@ def _setup_channel(self, grpc_port):
             log.info("Creating grpc channel for port: %s", grpc_port)
             self._channels[grpc_port] = grpc.insecure_channel("localhost:{}".format(grpc_port))
 
-    def _import_handlers(self):
+    def _import_handlers(self, model_name=None):
```

Comment:
Shall we pass `inference_script_path` instead of `model_name`?

Reply:
Sure I will make that change and update the PR.